src/git.md

```diff
@@ -73,6 +73,55 @@ the same, with some steps skipped:
  2. Make, stage, and commit your additional changes just like before.
  3. Push those changes to your fork: `git push`.
 
+## Troubleshooting git issues
+
+You don't need to clone `rust-lang/rust` from scratch if it's out of date!
+Even if you think you've messed it up beyond repair, there are ways to fix
+the git state that don't require downloading the whole repository again.
+Here are some common issues you might run into:
+
+### I deleted my fork on GitHub!
+
+This is not a problem from git's perspective. If you run `git remote -v`,
+it will say something like this:
+
+```
+$ git remote -v
+origin	https://github.com//rust-lang/rust (fetch)
+origin	https://github.com//rust-lang/rust (push)
+personal	https://github.com/jyn514/rust (fetch)
+personal	https://github.com/jyn514/rust (push)
+```
+
+You can change the URL of the fork like this:
```

Comment:
I think in most cases you will create a fork at the same URL. Maybe add something like
```suggestion
If you renamed your fork, you can change the URL of the fork like this:
```

Reply:
Right, yeah the only time this would be different is if you change your username I think. I updated the wording, thanks :)(